tests/README.md

```
@@ -1,105 +1,159 @@
+---
+
+## Test Marking: How to Mark Tests
+
+Markers are required for all tests. They are used for test selection in CI and local runs.
+
+### Marker Requirements
+- Every test must have at least one **Lifecycle** marker, and **test type** and **Hardware** markers.
+- **component** markers are required as applicable.
+
+### Marker Table
+| Category                | Marker(s)                | Description                        |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)